airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -274,7 +299,13 @@ def _update_override_params(self, o: dict):
                     self._update_container_yaml(container=container)
                 self._update_main_container_yaml(containers[0])
 
-                # adding env resources
+            # adding volume mounts from secrets which created before use the operator
+            if self.mount_files_from_secret:
+                o["spec"]["volumes"] = [
+                    *o["spec"].get("volumes", []),
+                    *[{"name": k, "secret": {"secretName": k}} for k, _ in self.mount_files_from_secret.items()],
```

Comment:
Please prepend the name with some autogenerated value, so the file mounted volumes will not clash with custom templates. e.g., if the template is:
``yaml
spec:
   volumes:
       my_secret: {}
```
and you add another, `my_secret` via the file load this would fail. (Duplicate value). So in this case we can add:
```yaml
                    *[{"name": "__auto_mounted_secret_files_"+k, "secret": {"secretName": k}} for k, _ in self.mount_files_from_secret.items()],
```

You can also make a function `_compose_auto_generated_file_mount_secret_name` which will make life easier if someone really wants to mess with it.

Reply:
It also needs to be unique (the secret name must be loaded only once :))